Flippers/build.gradle

```diff
@@ -1,17 +1,18 @@
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion "29.0.2"
+    compileSdkVersion 30
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 28
+        targetSdkVersion 30
         versionCode 1
         versionName "1.0"
     }
 }
 
 dependencies {
-    implementation "androidx.recyclerview:recyclerview:1.0.0"
+    implementation "androidx.recyclerview:recyclerview:1.1.0"
+    api "androidx.lifecycle:lifecycle-runtime-ktx:2.3.0"
```

Comment:
I think we shouldn't have `api` here and it should be used only in private modules or in such cases like ktx modules, which are in fact wrappers around java modules and it's expected to use `api` there 😊